tests/templates/kuttl/smoke/10-assert.yaml.j2

```diff
@@ -4,7 +4,7 @@
 ---
 apiVersion: kuttl.dev/v1beta1
 kind: TestAssert
-timeout: 600
+timeout: 120
```

Comment:
I also reduce this timeout when testing locally. But when executed on Jenkins, the deployment can take a while and I fear that we introduce test flakiness with this timeout.